Runtime/Game/LootLockerSDKManager.cs

```diff
@@ -4281,6 +4281,94 @@ public static void LootLockerPurchaseCatalogItems(string walletId, LootLockerCat
+        {
+            if (!CheckInitialized())
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerResponse>());
+                return;
+            }
+            var body = LootLockerJson.SerializeObject(new LootLockerRedeemAppleAppStorePurchaseForCharacterRequest()
+            {
+                transaction_id = transactionId,
+                character_id = characterId
+            });
+
+            LootLockerServerRequest.CallAPI(LootLockerEndPoints.redeemAppleAppStorePurchase.endPoint, LootLockerEndPoints.redeemAppleAppStorePurchase.httpMethod, body, onComplete: (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
+        }
+
+        /// <summary>
+        /// Redeem a purchase that was made successfully towards the Apple App Store for the current player
+        /// </summary>
+        /// <param name="productId">The id of the product that this redemption refers to</param>
+        /// <param name="purchaseToken">The token from the purchase successfully made towards the Google Play Store</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        public static void LootLockerRedeemGooglePlayStorePurchaseForPlayer(string productId, string purchaseToken, Action<LootLockerResponse> onComplete)
+        {
+            if (!CheckInitialized())
+            {
+                onComplete?.Invoke(LootLockerResponseFactory.SDKNotInitializedError<LootLockerResponse>());
+                return;
+            }
+            var body = LootLockerJson.SerializeObject(new LootLockerRedeemGooglePlayStorePurchaseForPlayerRequest()
+            {
+                product_id = productId,
+                purchase_token = purchaseToken
+            });
+
+            LootLockerServerRequest.CallAPI(LootLockerEndPoints.redeemGooglePlayStorePurchase.endPoint, LootLockerEndPoints.redeemGooglePlayStorePurchase.httpMethod, body, onComplete: (serverResponse) => { LootLockerResponse.Deserialize(onComplete, serverResponse); });
+        }
+
+        /// <summary>
+        /// Redeem a purchase that was made successfully towards the Apple App Store for a character that the current player owns
+        /// </summary>
+        /// <param name="productId">The id of the product that this redemption refers to</param>
+        /// <param name="purchaseToken">The token from the purchase successfully made towards the Google Play Store</param>
+        /// <param name="characterId">The id of the character to redeem this purchase for</param>
+        /// <param name="onComplete">onComplete Action for handling the response</param>
+        public static void LootLockerRedeemGooglePlayStorePurchaseForPlayer(string productId, string purchaseToken, int characterId, Action<LootLockerResponse> onComplete)
```

Comment:
I would name this LootLockerRedeemGooglePlayStorePurchaseForCharacter

Reply:
Fixed